### PR TITLE
data: theranamuskan

### PR DESCRIPTION
**Pull request by theranamuskan**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3671"></a>

### CODE REVIEW

File: data/RanaMuskan.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/theranamuskan/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/theranamuskan",
+      "icon": "FaGithub"
+    }
+  ]
```
Comment (ChinmayMhatre) [inline data/RanaMuskan.json:28]:
```suggestion
  ],
```
Missing comma

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Noticed a missing comma in the file

**Review by Tejender1521 — CHANGES_REQUESTED** [review]
Everything looks good except a comma [,] on line 28

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please rename your filename to your `username.json`, else some feature may not work for you!

It should be `theranamuskan.json`. Let us know if you need any help. 🛠

File: data/RanaMuskan.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Rana Muskan",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Contributor, Youtuber, love to code.",
+  "avatar": "https://github.com/RanaMuskan.png",
```
Comment (devvsakib) [inline data/RanaMuskan.json:6]: you can add tags like this;
image

**Review by devvsakib — CHANGES_REQUESTED** [review]
change your file name to your github username, 
`theranamuskan.json`

File: data/RanaMuskan.json (hunk 2 above)
Comment (theranamuskan) [reply data/RanaMuskan.json]: I have changed the code. Thanks 😊

File: data/theranamuskan.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      "url": "https://www.linkedin.com/in/theranamuskan/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/theranamuskan",
+      "icon": "FaGithub"
+    }
+  ],
+  "links": [
```
Comment (devvsakib) [inline data/theranamuskan.json:29]: you have duplicate code. delete lines 27 to 39

**Review by devvsakib — CHANGES_REQUESTED** [review]

File: data/theranamuskan.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Rana Muskan",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Contributor, Web Developer, Flutter Developer, Youtuber",
+  "avatar": "https://github.com/theranamuskan.png",
```
Comment (kumarsonsoff3) [inline data/theranamuskan.json:6]: As of our latest update, we no longer need the `avatar` key. So please remove this form your file. 
```suggestion
```

File: data/theranamuskan.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+   {
+      "name": "Follow on twitter",
+      "url": "https://twitter.com/theranamuskan",
+      "icon": "Fatwitter"
+   },
+   {
+     "name": "Follow me on Instagram",
+     "url": "https://www.instagram.com/theranamuskan/",
+     "icon": "FaInstagram"
+   }
```
Comment (kumarsonsoff3) [inline data/theranamuskan.json:39]: This part of your file contains duplicate elements. You can remove this one safely

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Hey @theranamuskan, a few more changes are required, then it should be good to go.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good, Thanks! 🎉

**Pull request merged**